dts/bindings/spi/nordic,nrf-spim.yaml

```diff
@@ -25,3 +25,28 @@ properties:
         Enables the workaround for the nRF52832 SoC SPIM PAN 58 anomaly.
         Must be used in conjunction with
         CONFIG_SOC_NRF52832_ALLOW_SPIM_DESPITE_PAN_58=y
+
+    has-rx-delay:
+      type: boolean
+      required: false
+      description: |
+        Indicates if the SPIM instance has the capability of delaying MISO
+        sampling. This property needs to be defined at SoC level DTS files.
```

Comment:
Please align this description with the existing language in the radio binding, again for consistency.